.eslintrc

```diff
@@ -26,5 +26,13 @@
         "@typescript-eslint/no-unused-vars": "off",
         "@typescript-eslint/no-empty-function": "warn",
         "no-inner-declarations": "off"
-    }
+    },
+    "overrides": [
+        {
+            "files": ["src/contracts/**/*"],
+            "rules": {
+                "@typescript-eslint/no-unused-vars": "off"
```

Comment:
Why did this change?

Reply:
because we were getting some unused variable errors because we are not using every method in the contract that is generated by the cli

Reply:
oh yeah, because you disabled that check for the whole repo 😄 https://github.com/wharfkit/token/commit/5f797c0bf6e8a9ef85145872fb7ecec78b1dbb64

Reply:
I think it's better to keep that check and just disable it for the contract file